CONTRIBUTORS.md

```diff
@@ -122,7 +122,7 @@ A networking library with helper functions around Asio and other network interfa
 Several CMake scripts came from other sources and were either used or modified for use in HELICS.
 
 - Lars Bilke [CodeCoverage.cmake](https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake)
-- NATIONAL HEART, LUNG, AND BLOOD INSTITUTE FindOctave.cmake
+- NATIONAL HEART, LUNG, AND BLOOD INSTITUTE
```

Comment:
Do we still need this line for another dependency?

Reply:
probably not